Classes/Neos/Neos/Ui/Domain/Model/Changes/AbstractCreate.php

```diff
@@ -176,6 +213,20 @@ protected function createNode(NodeInterface $parent)
             $node->setProperty($key, $value);
         }
 
+        if ($nodeType->isOfType('TYPO3.Neos:Content') && $this->getReferenceData()['subject']['fusionPath']) {
```

Comment:
actually, I think this condition is not fully correct:

1) additionally, the parent node must *always* be a ContentCollection for having this feedback work.

2) if the parent is NO contentCollection, we should reload the page using the feedback we already have.